x11-misc/xbattbar/xbattbar-1.4.9.ebuild

```
@@ -33,17 +32,11 @@ src_prepare() {
 	sed -i \
 		-e "s:usr/lib:usr/$(get_libdir):" \
 		xbattbar.c || die
-
-	tc-export PKG_CONFIG
 }
 
-src_compile() {
+src_configure() {
+	tc-export CC PKG_CONFIG
 	use kernel_linux && append-flags -Dlinux
```

```suggestion
	use kernel_linux && append-cppflags -Dlinux
```
and check that it's appearing on the compile-line

---

yeah, it does